src/Core/src/Platform/iOS/LayoutView.cs

```diff
@@ -37,11 +56,10 @@ public override void WillRemoveSubview(UIView uiview)
 				return null;
 			}
 
-			if (!result.UserInteractionEnabled)
+			if (Descendants is not null && Descendants.Contains(result) && !result.UserInteractionEnabled)
```

Comment:
Are we able to reproduce this behaviour without a webview? What happens if you add a view that has a wrapper - a label/image with a background color?

Is the `_UIRemoteView` view part of the UI tree?

How does this work when you are working with input transparency. There may be a case with a few nested controls and you tap one that is marked as InputTransparent. 

The `Descendants` list is just for the immediate children, but the webview is not that. So I think this will break other cases where you are intentionally "recreating" a similar situation with input transparency.

Reply:
I have applied changes to use the `SubViews ` property. Each LayoutView will check its direct children. In case any element has children that we must consider, it will be a LayoutView that will also be in charge of check its direct children.